README.md

```diff
@@ -8,17 +8,36 @@ Liqo is a framework to enable dynamic sharing across Kubernetes Clusters.
 
 Liqo can be installed via Helm. 
 
+### Pre-requisites
+
+* Two k8s/k3s clusters
+* Supported CNIs:
+   * Flannel
+   * Calico
+* Helm 3   
+
+### Pre-Installation
+
+So far, the only way to add a cluster is to rely on explicit forgery of a kubeconfig. 
+
+### Installation
+
+
+
 ```
 cd deployments/liqo_chart/
 helm dep up
 helm install -n liqo liqo ./ -f values.yaml
 ```
 
```

Comment:
Once we install Liqo, what do we do?
We should add some instructions to tell people (1) how to test that everything looks correct, (2) how to use it.